### DISCUSSION TITLE
Can't fountd COM1 in XIAO_ESP32C3，need com1

### DISCUSSION THREAD

**Discussion by pt2go** [nanoframework/Home#1321] [General]
### Description

sample SerialCommunication log：
Available ports: 
 COM2

System Information
HAL build info: nanoCLR running @ ESP32_C3 built with ESP-IDF e8bdaf9
  Target:   XIAO_ESP32C3
  Platform: ESP32

Firmware build Info:
  Date:        Jun  6 2023
  Type:        MinSizeRel build, chip rev. >= 3, without support for PSRAM
  CLR Version: 1.8.1.307
  Compiler:    GNU ARM GCC v8.4.0

### How to solve the problem

need to add com1 to FW

**Comment by josesimoes** [comment]
@pt2go I'm moving this to conversations. 

As stated in the initial steps that you can read when opening an issue, we are using issues to track confirmed bugs and pending work. 

This is a question that can be answered in a short conversation. 😉

**Comment by josesimoes** [comment]
COM1 is used by Wire Protocol to communicate with Visual Studio for debugging and device discovery. 
That's the reason why is not available

**Comment by pt2go** [comment]
Thank you for your answer. XIAO_ ESP32C3 uses USB for download and debugging, so COM1 should be able to be released for use. So i raised the feature request

**Comment by josesimoes** [comment]
@pt2go you're better using the dev container for a local build.
How far have you gone in your attempt?

**Comment by matsujirushi** [comment]
Hi @pt2go ,

Serial ports of Seeed Studio XIAO ESP32C3 are:
1. USB_SERIAL_JTAG (USB-C)
2. UART (D6 and D7 pin)

No.1 is used for communication with Visual Studio, so it cannot be used from the application. =[WireProtocol](https://github.com/nanoframework/Home/discussions/1321#discussioncomment-6109233)
No.2 is available from the application. =COM2 ([example](https://zenn.dev/matsujirushi/articles/96803dd819e4bc))

Therefore, applications can only see COM2.

Is this the answer?
